test-integ/peering_commontopo/sharedtopology_test.go

```diff
@@ -0,0 +1,42 @@
+package peering
+
+import (
+	"testing"
+)
+
+// Tests compatible with commonTopo should implement commonTopoSuite and be added
+// to the commonTopoSuites slice inside.
+func TestSuitesOnSharedTopo(t *testing.T) {
+	if !*commonTopologyFlag {
```

Comment:
By default, this flag is `false`. But I think by default we *should* be using shared topo. We should optimize for CI first, then local devs second.